client-failover/client-failover-sample-hazelcast5/src/main/java/sample/com/hazelcast/clientfailover/ClientWithSsl.java

```diff
@@ -0,0 +1,127 @@
+package sample.com.hazelcast.clientfailover;
+
+import java.util.Properties;
+import java.util.Random;
+
+import com.hazelcast.client.HazelcastClient;
+import com.hazelcast.client.config.ClientConfig;
+import com.hazelcast.client.config.ClientFailoverConfig; // <1>
+import com.hazelcast.config.SSLConfig;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.core.HazelcastJsonValue;
+import com.hazelcast.map.IMap;
+import com.hazelcast.sql.SqlResult;
+import com.hazelcast.sql.SqlRow;
+import com.hazelcast.sql.SqlService;
+import sample.com.hazelcast.clientfailover.model.City;
+import sample.com.hazelcast.clientfailover.model.Country;
+
+
+// tag::intro-comments[]
+/**
+ * This is a boilerplate application that configures a client 
+ * to connect to a Hazelcast Viridian cluster.
+ * See: https://docs.hazelcast.com/cloud/get-started
+ * 
+ * The numbers in angled brackets are used in the documentation 
+ * to call out and explain important steps.
+ * 
+ * The comments marked tag:: are also used to include and exclude 
+ * snippets in the docs.
+ */
+// end::intro-comments[]
+
+public class ClientWithSsl {
+
+    public static void main(String[] args) throws Exception {
+
+        ClientFailoverConfig clientFailoverConfig = new ClientFailoverConfig(); // <2>
+        clientFailoverConfig.addClientConfig(getPrimaryClientConfig());
+        clientFailoverConfig.addClientConfig(getSecondaryClientConfig());
+
+        HazelcastInstance client = HazelcastClient.newHazelcastFailoverClient(clientFailoverConfig);
+
+        System.out.println("Connection Successful!");
+
+        nonStopMapExample(client); // <3>
+
+        client.shutdown();
+
+        System.exit(0);
+
+    }
+
+    // <4>
+    private static ClientConfig getPrimaryClientConfig() throws Exception {
+        ClassLoader classLoader = ClientWithSsl.class.getClassLoader();
+        Properties props = new Properties();
+        props.setProperty("javax.net.ssl.keyStore", classLoader.getResource("client.keystore").toURI().getPath());
+        props.setProperty("javax.net.ssl.keyStorePassword", "YOUR_KEYSTORE_PASSWORD");
+        props.setProperty("javax.net.ssl.trustStore",
+            classLoader.getResource("client.truststore").toURI().getPath());
+        props.setProperty("javax.net.ssl.trustStorePassword", "YOUR_KEYSTORE_PASSWORD");
+        ClientConfig config = new ClientConfig();
+        config.getNetworkConfig().setRedoOperation(true);
+        config.getNetworkConfig().setSSLConfig(new SSLConfig().setEnabled(true).setProperties(props));
+        config.getNetworkConfig().getCloudConfig()
+            .setDiscoveryToken("YOUR_DISCOVERY_TOKEN")
+            .setEnabled(true);
+        config.setProperty("hazelcast.client.cloud.url", "YOUR_CLIENT_URL");
```

Comment:
I think we can remove this property from the code samples. As far as I know, the cloud team is using it for testing purposes, but I don't see the point of exposing it to the user.

Reply:
@mdumandag. Just tried removing this from the code and it no longer runs as it can't seem to find the cluster. Do we need to provide a hard-coded value?